CentralisedPackageConverter/PackageConverter.cs

```diff
@@ -397,13 +401,28 @@ private void ConvertProject(FileInfo csprojFile, bool dryRun, Encoding encoding,
 
             if (!versioning.IgnorePackageVersion(version))
             {
-                Output.InfoLine(" Found new reference: {0} {1} with Condition {2}",
-                    package, version.ToFullString(), condition);
+                if (condition != String.Empty)
+                {
+                    Output.InfoLine(" Found new reference: {0} {1} with Condition {2}",
+                        package, version.ToFullString(), condition);
+                }
+                else
+                {
+                    Output.InfoLine(" Found new reference: {0} {1}",
+                    package, version.ToFullString());
+                }

```

Comment:
Did some simple if/elses here to remove dangling string formats for cases where there were no conditions.